### ISSUE TITLE
.find() multiple items by their keys doesn't work anymore

### ISSUE THREAD

**Issue by tyrghen** [balderdashy/sails-redis#58]
.find() with multiple keys as criteria doesn't work anymore, it defaults all find() queries to findOne() when you're using the primary key in the where clause, regardless of the fact that it is an array of values.

I explained it on the commit below:
https://github.com/balderdashy/sails-redis/commit/5d46db1fcad6b25a868b51651508ae38a67e3f00

That change did effectively fix the find() by primary key but it lacks a check for an array of values and thus breaks the find command when passing multiple keys.
```javascript
MyCollection.find(['key1','key2']).exec(...)
```

The following line:
```javascript
if(criteria.where && Utils.present(criteria.where[primary])) {
```
https://github.com/balderdashy/sails-redis/blob/master/lib/database/index.js#L184

should check that the clause value is not an array:
```javascript
if(criteria.where && Utils.present(criteria.where[primary]) && !Array.isArray(criteria.where[primary])) {
```

I was unsure if I should only comment on the commit itself or open an issue here, I'm relatively new to Github... sorry if that wasn't the right way to do it.

**Comment by cha0s** [comment]
Can confirm this is broken and the change fixes it.

**Comment by ralph-rush** [comment]
https://github.com/ralph-rush/sails-redis/commit/53d7f4e584c9e40a210265b6e8c5d0368afe156d

**Comment by WesRateur** [comment]
Can confirm too the change fixes it. Save my day thanks @cha0s

**Comment by Ryanc1256** [comment]
Should be fixed with Pull request #67
